scan-triggers/aws-typescript-bucket-full-scan/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "fss-full-scan",
+  "version": "v1.0.0",
+  "author": {
+    "name" : "Raphael Bottino",
+    "email" : "raphabot@gmail.com",
+    "url" : "https://github.com/raphabot/"
+  },
```

Comment:
It would be better if the author info is changed to Cloud One File Storage Security. We are more than thankful to your contribution.